DOC-5266 Aling data migration tool summaries, articles, and attributes (#202)
datastax/migration-docs: modules/ROOT/pages/contributions.adoc

{company} {product} ({product-short}) provides a simple and reliable way for users to migrate an existing {cass-reg} or {dse} cluster to {astra-db}, or to any {cass-short} or {dse-short} cluster, without any interruption of service to the client applications and data.

> {product-proxy} is open source software (OSS). We welcome contributions from the developer community via Pull Requests on a fork, for evaluation by the {product-short} team.

Reviewer: ```suggestion
{product-proxy} is open source software (OSS).
{company} welcomes contributions from the developer community through Pull Requests on a fork, for evaluation by the {product-short} team.
```

Reviewer: GDDSG recommends against using _via_
https://developers.google.com/style/word-list#via

Author: The whole page is like this. I have a ticket for it up next.
